app/src/pages/inside/testCaseLibraryPage/createTestCaseModal/testCaseDetails/steps/steps.scss

```diff
@@ -64,3 +104,46 @@
     height: 24px;
   }
 }
+
+@keyframes dropHighlight {
+  0% {
+    box-shadow: 0 0 0 3px rgba($COLOR--topaz-focused, 0.4);
+    transform: scale(1.01);
+  }
+
+  50% {
+    box-shadow: 0 0 0 6px rgba($COLOR--topaz-focused, 0.2);
+  }
+
+  100% {
+    box-shadow: 0 0 0 0 rgba($COLOR--topaz-focused, 0);
+    transform: scale(1);
+  }
+}
+
+.drag-preview {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 8px;
+  width: 376px;
+  height: 52px;
+  padding-left: 12px;
+  background-color: var(--rp-ui-base-bg-000);
+  border: 1px solid $COLOR--e-200;
+  border-radius: 4px;
+  box-shadow: 0 4px 12px rgba(0, 0, 0, 0.15);
+  font-size: 13px;
```

Comment:
is it possible to use `@font-scale` mixin here?

Reply:
I see we have a such mixin in ui-kit repo, but we do not export it and it's impossible. We can forward it, but it need to update packa.json and dist folder. I think if we want it, it should be done not for this PR, but in tech task. 

cc: @siarheirazuvalau